volatility3/framework/plugins/windows/indirect_system_calls.py

```diff
@@ -0,0 +1,124 @@
+# This file is Copyright 2024 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import struct
+import logging
+from typing import List, Optional
+
+import capstone
```

Comment:
Again, `ImportError` protection would be handy here, so it fails gracefully letting people know it can't operate.  I believe the yara plugins already do something similar?

Reply:
done. We should add to requirements.txt for parity release given how many plugins need it.

Reply:
Thanks, it is already in `requirements.txt`, see below.